doc/manuals/devel/mongoBackend.md

```diff
@@ -580,6 +582,65 @@ _MB-21: mongoUnsubscribeContextAvailability_
 
 [Top](#top)
 
+#### `mongoRegistrationGet` (SR2)
+
+`mongoRegistrationGet` encapsulates the logic for getting context registrations for NGSIv2 API.
+
+The header file contains two functions:
+
+* `mongoRegistrationGet()`, to get individual context registrations by id, and
+* `mongoRegistrationsGet()`, to get all context registrations.
+
+They both return a `Registration` object (or a vector of `Registration` objects, in the case of get all) with the result.
+
+In both cases, the implementation is based on a query on the `registrations` collection, ([described as part of the database model in the administration documentation](../admin/database_model.md#registrations-collection)).
+
+Regarding `mongoRegistrationGet()`:
+
+<a name="flow-mb-23"></a>
+![mongoRegistrationGet](images/Flow-MB-23.png)
+
+_MB-23: mongoRegistrationGet_
+
+* `mongoRegistrationGet()` is invoked from a service routine (step 1).
+* Depending on `-reqMutexPolicy`, the request semaphore may be taken (read mode) (step 2). See [this document for details](semaphores.md#mongo-request-semaphore). 
+* The registration is retrieved from the database using `collectionQuery()` in the `connectionOperations` module (steps 3 and 4).
+* Several `set*()` functions are used in order to fill the `Registration` object to return.
+* If the request semaphore was taken in step 2, then it is released before returning (step 5). 
+
+Regarding `mongoRegistrationsGet()`:
+
+<a name="flow-mb-24"></a>
+![mongoRegistrationsGet](images/Flow-MB-24.png)
+
+_MB-24: mongoRegistrationsGet_
+
+* `mongoRegistrationsGet()` is invoked from a service routine (step 1).
+* Depending on `-reqMutexPolicy`, the request semaphore may be taken (read mode) (step 2). See [this document for details](semaphores.md#mongo-request-semaphore). 
+* The subscription is retrieved from the database using `collectionRangedQuery()` in the `connectionOperations` module (steps 3 and 4).
+* For each subscription to return, several `set*()` functions are used in order to fill the `Registration` objects. 

```

Comment:
For each subscription to return ...   "s/subscription/registration/" ?

Reply:
I'll fix

Reply:
Fixed in d4bc65f